test/unit/commands/network-specs.js

```diff
@@ -149,6 +149,18 @@ describe('Network', function () {
       await driver.setGeoLocation('location').should.become('res');
     });
   });
+  describe('getGeoLocation', function () {
+    it('should get location', async function () {
+      adb.getGeoLocation.returns({
+        latitude: '1.1',
+        longitude: '2.2',
+      });
+      const {latitude, longitude, altitude} = await driver.getGeoLocation();
+      isNaN(latitude).should.be.false;
+      isNaN(longitude).should.be.false;
+      isNaN(altitude).should.be.false;
```

Comment:
I think we should be checking if these are numbers. This will be a false-positive if latitude, longitude or altitude is `undefined` or `null`.

Reply:
how about Number.isNaN?

Reply:
Yeah I can't find anything for checking that the type is a number for chai-as-promised.

This should be okay as-is.